src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/ExecutionServiceCluster.scala

```diff
@@ -0,0 +1,67 @@
+package org.broadinstitute.dsde.rawls.dataaccess
+
+import org.broadinstitute.dsde.rawls.RawlsException
+import org.broadinstitute.dsde.rawls.dataaccess.slick.WorkflowRecord
+import org.broadinstitute.dsde.rawls.model._
+
+import scala.concurrent.{Future}
+import scala.util.Try
+
+/**
+  * Created by davidan on 6/14/16.
+  */
+trait ExecutionServiceCluster extends ErrorReportable {
+  override val errorReportSource = "cromwell"
+
+  // ====================
+  // facade methods
+  // ====================
+  def submitWorkflow(wdl: String, inputs: String, options: Option[String], userInfo: UserInfo): Future[ExecutionServiceStatus]
+
+  def submitWorkflows(workflowRecs: Seq[WorkflowRecord], wdl: String, inputs: Seq[String], options: Option[String], userInfo: UserInfo): Future[(ExecutionServiceId, Seq[Either[ExecutionServiceStatus, ExecutionServiceFailure]])]
+
+  def status(wfe: WorkflowExecution, userInfo: UserInfo): Future[ExecutionServiceStatus]
+
+  def callLevelMetadata(wfe: WorkflowExecution, userInfo: UserInfo): Future[ExecutionMetadata]
+
+  def outputs(wfe: WorkflowExecution, userInfo: UserInfo): Future[ExecutionServiceOutputs]
+
+  def logs(wfe: WorkflowExecution, userInfo: UserInfo): Future[ExecutionServiceLogs]
+
+  def abort(wfe: WorkflowExecution, userInfo: UserInfo): Future[Try[ExecutionServiceStatus]]
+
+}
+
+object ExecutionServiceCluster {
+  // this should match the value in liquibase for pre-sharded workflows
+  val DEFAULT_KEY = ExecutionServiceId("default")
+}
+
+class WorkflowExecution(val id: String, val executionServiceId: ExecutionServiceId) {
```

Comment:
why do we need this class anymore? I think it only adds confusion: what is a workflow vs. workflow record vs. workflow execution? How is it used where a workflow record would not do just as well?